### ISSUE TITLE
Update PDFgui user guide

### ISSUE THREAD

**Issue by Tieqiong** [diffpy/diffpy.pdfgui#167]
PDFgui user guide has not been updated since 2.0 release (May 2023). We might want this to be deployed along with the api document. There are also content that need to be updated (e.g. things that related to setup.py which we replace with pyproject.toml).

The PDFgui user guide was made using texi2html 5.0 and deployed on https://www.diffpy.org/doc/pdfgui/index.html. This is also not aligned with the new deploy namespace (which it should be something like https://www.diffpy.org/pdfgui/...). Maybe we should change this and delete the older resource. 

@sbillinge I also found out that the new vs. old document namespace is also happening in other packages too. For example we deployed the new diffpy.structure api on https://www.diffpy.org/diffpy.structure/, but using similar pattern using the about pdfgui link I also found the older api documents on https://www.diffpy.org/doc/structure/, which is showing 3.1.0 as the latest version (which should be 3.2.0 now).

**Comment by sbillinge** [comment]
yes, I would like to get everything synchronized here and following the new approach, whatever that is.

**Comment by Tieqiong** [comment]
Things that need updates:
1. package related info
    - release number and date
    - license information under Availability section (I'm not sure if PDFfit2 and PDFgui are under BSD license as mentioned in the manual)
    - update installation section to align with the cookiecutted style readme
    - update news (what is new section)
    - community (do we want diffpy-dev Google groups?)
2. Content
    - figure 3.6 and 3.7 (The opened file is different from the file mentioned in the text)
    - API documentation link (used to be on danse.us site, see #186 )
    - all the `texi2html` related things and generating date. We might want to make changes on the texinfo files first and try using this tool to generate html.

@sbillinge please add anything that we might want to change.

**Comment by sbillinge** [comment]
> Things that need updates:
> 
> 1. package related info
>    
>    * release number and date
>    * license information under Availability section (I'm not sure if PDFfit2 and PDFgui are under BSD license as mentioned in the manual)
They are not, please refer tests to the actual license
>    * update installation section to align with the cookiecutted style readme
Yes, good
>    * update news (what is new section)
Yes
>    * community (do we want diffpy-dev Google groups?)
No, put everything through dugout users or GitHub issues. I want to EOL dugout-dev.  Maybe we can deprecate it?
> 2. Content
>    
>    * figure 3.6 and 3.7 (The opened file is different from the file mentioned in the text)
>    * API documentation link (used to be on danse.us site, see [Replace DANSE links #186](https://github.com/diffpy/diffpy.pdfgui/issues/186) )
Now it will be at GitHub.io.  I want to deprecate docs at diffpy.org too.
>    * all the `texi2html` related things and generating date. We might want to make changes on the texinfo files first and try using this tool to generate html.
Yes, not sure about this, let's discuss.
> 
> @sbillinge please add anything that we might want to change.

**Comment by sbillinge** [comment]
Per conversation during release on 2025-01-14, we need something in the release workflow that either builds and deploys the manual, or simply presents instructions to the releaser to remind them to do it locally before doing a full release

**Comment by sbillinge** [comment]
@sbillinge working on this.  Hope to complete it on Friday.
